examples/listbox/listbox-collapsible.html

```diff
@@ -28,6 +28,9 @@
   </nav>
   <main>
   <h1>Collapsible Dropdown Listbox Example</h1>
+  <div class="advisement">
+    <p><strong>DEPRECATION WARNING:</strong> this pattern has been deprecated, and will be removed in a future version of the Authoring Practices. The <a href="../combobox/combobox-select-only.html">select-only combobox</a> should be used as an alternative to this pattern.</p>
```

Comment:
I would add ARIA (or WAI-ARIA) as follows:

```suggestion
    <p><strong>DEPRECATION WARNING:</strong> this pattern has been deprecated, and will be removed in a future version of the ARIA Authoring Practices. The <a href="../combobox/combobox-select-only.html">select-only combobox</a> should be used as an alternative to this pattern.</p>
```